gwpy/table/io/pycbc.py

```diff
@@ -234,11 +234,21 @@ def empty_hdf5_file(h5f, ifo=None):
     # the decorator opens the HDF5 file for us, so h5f is guaranteed to
     # be an h5py.Group object
     h5f = h5f.file
+    # if the root group is empty, the file is empty
     if list(h5f) == []:
         return True
-    if ifo is not None and (ifo not in h5f or list(h5f[ifo]) == ['psd']):
-        return True
-    return False
+
+    # for each group (or the IFO group given by the user),
+    # check whether there is any useful content
+    if ifo is None:
+        groups = h5f.keys()
+    else:
+        groups = [ifo]
+    for group in groups:
+        print(set(h5f.get(group, [])))
```

Comment:
@duncanmmacleod, these changes look great, unit tests are passing, and hotfixing them fixes the issues I was seeing on the production summary pages, so thank you for putting this together.

That having been said, I think this stray print statement needs to be removed, after which I'm happy to approve this.